src/enclave_proc/resource_manager.rs

```diff
@@ -905,6 +922,189 @@ impl EnclaveManager {
         Ok(())
     }
 
+    /// Set measurements field inside EnclaveHandle
+    pub fn set_measurements(
+        &mut self,
+        measurements: BTreeMap<String, String>,
+    ) -> NitroCliResult<()> {
+        self.enclave_handle
+            .lock()
+            .map_err(|e| {
+                new_nitro_cli_failure!(
+                    &format!("Failed to acquire lock: {:?}", e),
+                    NitroCliErrorEnum::LockAcquireFailure
+                )
+            })?
+            .measurements = measurements;
+        Ok(())
+    }
+
+    /// Initialize a Sha384 hasher struct
+    pub fn init_sha384_hasher(hasher_name: &str) -> NitroCliResult<EifHasher<Sha384>> {
```

Comment:
I simply don't understand why are this methods of the EnclaveManager at best they could be just some utility functions in the EifHasher module or in the utils module.